src/Selenium2Library/keywords/_element.py

```diff
@@ -267,13 +267,15 @@ def element_text_should_be(self, locator, expected, message=''):
                           "in fact it was '%s'." % (locator, expected, actual)
             raise AssertionError(message)
 
-    def get_element_attribute(self, attribute_locator):
+    def get_element_attribute(self, locator, attribute_name=None):
         """Return value of element attribute.
 
-        `attribute_locator` consists of element locator followed by an @ sign
+        Typically, `locator` consists of element locator followed by an @ sign
         and attribute name, for example "element_id@class".
+        In case when `attribute_name` is not None, `locator` is used only to get webelement.
         """
-        locator, attribute_name = self._parse_attribute_locator(attribute_locator)
+        if attribute_name is None:
```

Comment:
Doing the review over mobile and therefore can not be totally sure about the line numbers. But if it's within same method, then feel free to change it. If it's not then a separate pull request.